api/nodes/serializers.py

```diff
@@ -110,6 +118,7 @@ def get_pointers_count(self, obj):
         return len(obj.nodes_pointer)
 
     def create(self, validated_data):
+        validated_data.pop('type')
```

Comment:
Discussed in person: Removing 'id' and 'type' from the validated data should happen in the base class (JSONAPISerializer).